database/migrations/2020_03_18_173706_lti_connection_content_id_rename.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class LtiConnectionContentIdRename extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('lti_connections', function (Blueprint $table) {
+            $table->renameColumn('lti_content_id', 'content_id');
```

Comment:
You should modify the previous migration instead !  I suppose it has not been deployed to prod yet (as it has not been merged, to the main branch, there is no reason for that)

Reply:
done